src/settings-ui/Settings.UI/Views/AlwaysOnTopPage.xaml

```diff
@@ -1,141 +1,148 @@
-                            </controls:Setting>
-                        </controls:SettingExpander.Header>
-                        <controls:SettingExpander.Content>
-                            <StackPanel>
-
-                                <controls:Setting x:Uid="AlwaysOnTop_FrameColor_Mode" Style="{StaticResource ExpanderContentSettingStyle}" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}">
-                                    <controls:Setting.ActionContent>
-                                        <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource BoolToComboBoxIndexConverter}}" MinWidth="{StaticResource SettingActionControlMinWidth}">
-                                            <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Custom_Color"/>
-                                            <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Windows_Default"/>
-                                        </ComboBox>
-                                    </controls:Setting.ActionContent>
-                                </controls:Setting>
+                <controls:SettingsGroup
+                    x:Uid="AlwaysOnTop_Behavior_GroupSettings"
+                    IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
+                    <labs:SettingsExpander
+                        x:Uid="AlwaysOnTop_FrameEnabled"
+                        HeaderIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xEB3C;}"
+                        IsExpanded="True">
+                        <ToggleSwitch
+                            x:Uid="ToggleSwitch"
+                            IsOn="{x:Bind Mode=TwoWay, Path=ViewModel.FrameEnabled}" />
+                        <labs:SettingsExpander.Items>
+                            <labs:SettingsCard
+                                x:Uid="AlwaysOnTop_FrameColor_Mode"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}">
+                                <ComboBox
+                                    MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                    SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource BoolToComboBoxIndexConverter}}">
+                                    <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Custom_Color" />
+                                    <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Windows_Default" />
+                                </ComboBox>
+                            </labs:SettingsCard>
+                            <labs:SettingsCard
+                                x:Uid="AlwaysOnTop_FrameColor"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource ReverseBoolToVisibilityConverter}}">
```

Comment:
This converter is hiding the controls.
`True = 0 = custom`. but `TrueValue="Collapsed"`
(off-topic idea: we _could_ always leave the color-picker visible and paint it with either the custom or windows color)

Reply:
This didn't change with the PR, right?

Reply:
Hm, looks like you're right. The names changed, but it should work.